cloud/gcp-functions/cmd/orphandResources.go

```diff
@@ -0,0 +1,81 @@
+package orphandResources
+
+import (
+        "log"
+        "fmt"
+        "net/http"
+        "context"
+        "os"
+        "strings"
+
+        "google.golang.org/api/compute/v1"
+)
+
+func CleanOrphandResources(w http.ResponseWriter, r *http.Request) {
+
+    ctx := context.Background()
+    computeService, err := compute.NewService(ctx)
+    if err != nil {
+        log.Fatal("Error: create service", err)
+    }
+
+    project := os.Getenv("GCP_DEV_PROJECT")
+    targetPoolAggregatedList, err := computeService.TargetPools.AggregatedList(project).Do()
+
+    if err != nil {
+            log.Fatal("Error: get targetPoolAggregatedList", err)
+    }
+
+    toDelete := false
+    for _, targetPoolList := range targetPoolAggregatedList.Items {
+        for _, targetPoolItem := range targetPoolList.TargetPools {
+            zone := strings.Split(targetPoolItem.Instances[0], "/")[8]
+            instanceName := strings.Split(targetPoolItem.Instances[0], "/")[10]
+            region := strings.Split(targetPoolItem.Region, "/")[8]
+
+            if len(targetPoolItem.Instances) > 0 {
+                _, err := computeService.Instances.Get(project, zone , instanceName).Context(ctx).Do()
```

Comment:
space after zone and comma

Reply:
this and Tomislav's other comment would be fixed by running `go fmt` on this file